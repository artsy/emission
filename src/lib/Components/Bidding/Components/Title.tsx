import React from "react"
import { TextProperties } from "react-native"
import styled from "styled-components/native"

import { Fonts } from "../../../data/fonts"

export class Title extends React.Component<TextProperties> {
  render() {
    return <StyledText {...this.props}>{this.props.children}</StyledText>
  }
}

const StyledText = styled.Text`
  font-family: "${Fonts.GaramondRegular}";
  margin: 10px;
  font-weight: bold;
  font-size: 20px;
  text-align: center;
`
