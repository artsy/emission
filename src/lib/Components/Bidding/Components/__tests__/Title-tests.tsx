import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"

import { Title } from "../Title"

it("renders properly", () => {
  const bg = renderer.create(<Title>Confirm your bid</Title>).toJSON()
  expect(bg).toMatchSnapshot()
})
