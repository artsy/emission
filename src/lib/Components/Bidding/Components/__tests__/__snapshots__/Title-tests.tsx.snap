// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<Text
  className="sc-bwzfXH uTxCW"
  fontFamily="AGaramondPro-Semibold"
  fontSize={22}
  lineHeight={28}
  m={4}
  style={
    Array [
      Object {
        "fontFamily": "AGaramondPro-Semibold",
        "fontSize": 22,
        "lineHeight": 28,
        "marginBottom": 32,
        "marginLeft": 32,
        "marginRight": 32,
        "marginTop": 32,
        "textAlign": "center",
      },
      undefined,
    ]
  }
  textAlign="center"
>
  Confirm your bid
</Text>
`;
